lib/jekyll/commands/new.rb

```diff
@@ -153,8 +153,11 @@ def after_install(path, options = {})
         def bundle_install(path)
           Jekyll.logger.info "Running bundle install in #{path.cyan}..."
           Dir.chdir(path) do
-            exe = Gem.bin_path("bundler", "bundle")
-            process, output = Jekyll::Utils::Exec.run("ruby", exe, "install")
+            gem "bundler"
```

Comment:
What is this line supposed to do..?

Reply:
`gem "bundler"` activates a gem and puts it into the `loaded_specs`.
For more details, please refer to the source code of this method.
https://rubydocs.org/d/ruby-2-6-2/classes/Kernel.html#method-i-gem

Reply:
Nice.. I had no idea that private instance methods were documented as well.
Could you please add a comment above this line of code to help future readers refer to the docs as well.
```ruby
# Activate 'bundler' gem and puts it into the `loaded_specs`.
# For details: https://rubydocs.org/d/ruby-2-4-0/classes/Kernel.html#method-i-gem
gem "bundler"
```